index.html

```diff
@@ -0,0 +1,139 @@
+        <li><img class="fb" src="asset/facebook.png" alt="facebook"></li>
+        <li><img class="twit" src="asset/twitter.png" alt="twitter"></li>
+        <li>English</li>
+        <li>My page</li>
+        <li>Log out</li>
+        </ul>
+    </div>
+    <div class="desk-menu">
+        <img class="mic-logo" src="asset/mic logo.png">
+        <ul class="desk-list">
+            <li onclick="openmenu()" class="ab-desk"><a href="./about.html">About</a></li>
+            <li onclick="openmenu()" class=""><a href="#pro">Program</a></li>
+            <li onclick="openmenu()"><a href="#"></a>Join</li>
+            <li>Sponsor</li>
+            <li>News</li>
+            <li class="desk-cc">CC Campaign</li>
+        </ul>
+    </div>
+    <div class="toggle">
+        <img onclick="openMenu()" id="hamburger"  src="asset/mobile_menu 1.png">
+        <img class="mic-log" src="asset/mic logo.png">
+    </div>
+    <div class="menu-mob hidden" id="mobile-menu">
+        <img  onclick="closeMenu()" class="closing" src="asset/Icon - Cancel.png" id="closing-icon">
+
+        <ul class="menu-list" id="hum-list">
+            <li onclick="openMenu()" class=""><a class="" href="./about.html">About</a></li>
+            <li onclick="openmenu()" class=""><a class="" href="#program"></a>Program</li>
+            <li>Join</li>
+            <li>Sponsor</li>
+            <li>News</li>
+            <li>CC Campaign</li>
+        </ul>
+    </div>
+    </header>
+    <section class="main-section">
+        <div class="section-head">
+        <h1 class="greeting">"Hello! Sharing World"</h1>
+        <h2 class="topic">Research Conference and Seminar at MIC <br> 2023</h2>
+    </div>
+        <p class="detail">The seminars contribute to professional development, dissemination of research findings, and the advancement of knowledge within the respective field..
+        </p>
+        <div class="main-bottom">
+        <h3 class="time-fram">  2023.8.8(THU) ~ 16(WED)</h3>
+    <p class="address">@Maryland college hall Addis Ababa,Ethiopia</p>
+</div>
+    </section>
+    <section class="program" id="pro">
+        <div class="m-p">
+        <h1 class="m-pro">Main Program</h1>
+        <hr class="uderline"></hr>
+    </div>
+    <div class="pro-list">
+        <div class="lecture">
+            <img src="asset/lecture icon.png" alt="">
+            <p class="p-list">Lecture</p>
+            <p class="p-det">Listen to the speakers from various university.</p>        
+        </div>
+        <div class="Exhibition">
+            <img src="asset/CC icon.png" alt="">
+            <p class="p-list">CC Exhibition</p>
+            <p class="p-det">Appreciate various findings.</p>
+        </div>
+        <div class="forum">
+            <img src="asset/forum icon.png" alt="">
+            <p class="p-list">Forum</p>
+            <p class="p-det">Have the time to share your thoughts and opinions with scholars.
+            </p>
+        </div>
+        <div class="workshop">
```

Comment:
- [ ] Great job so far. please kindly ensure the card sizes are the same like in the original version. kindly add some margin left and righ to them.
- [ ] kindly note the color on the page is not red. its \#EC5242.. please kindly fix that including those on the red small lines.
yours
![Screenshot (1290)_LI](https://github.com/Surafels/Capstone_Project/assets/65298048/1d71a23d-e582-4c32-a629-3537b5258684)
behance
![Screenshot (1289)](https://github.com/Surafels/Capstone_Project/assets/65298048/4770c82c-7f2f-4eb9-8252-cf18576c69e1)